src/Automation/Abstractions/ISystemDateTime.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Abstractions
+{
+    internal interface ISystemDateTime
```

Comment:
>internal [](start = 4, length = 8)

I'd love to see some /// comments to capture the intent of these classes that are being used to inject seams

Reply:
I'm thinking perhaps such classes belong in their own project whose intent is clear what they're for. I didn't have time to do that in this rev.